sources/packages/api/src/database/entities/designation-agreement-status.type.ts

```diff
@@ -17,4 +17,12 @@ export enum DesignationAgreementStatus {
    * was declined by the Ministry.
    */
   Declined = "Declined",
+  /**
+   * The designation agreement status is designated
+   */
+  Designated = "Designated",
```

Comment:
This enum represents the DB enum and `Designated` and `Not designated` do not belong here. Please create a specific enum for this at the service layer.